arrays/arrays.c

```diff
@@ -22,10 +22,14 @@ typedef struct Array
 Array *create_array(int capacity)
 {
   // Allocate memory for the Array struct
+  Array *arr = malloc(sizeof(Array));
 
   // Set initial values for capacity and count
+  arr->capacity = capacity;
+  arr->count = 0;
 
   // Allocate memory for elements
+  arr->elements = malloc(capacity * sizeof(char *));
```

Comment:
Have you considered using calloc by any chance?, definitely looking the differences as they can make a difference in the code efficiency, other than that great work!